matguide.py

```diff
@@ -71,20 +73,20 @@ def __init__(self,  parent=None):
         self.maleBtn.clicked.connect(self.getMat)
         self.femaleBtn.clicked.connect(self.getMat)
 
-        screen=QDesktopWidget().screenGeometry()
-        window=self.geometry()
-        self.setGeometry((screen.width()-window.width())/2, self.settings['WindowAnchor']-
-                window.height(), window.width(), window.height())
-        self.setMinimumSize(window.width(), window.height())
-        self.setMaximumSize(window.width(), window.height())
+       # screen=QDesktop().screenGeometry()
```

Comment:
This is not supported in the new pyqt library so commented it out, not sure if it's needed and I need to update this code or I can remove it entirely.